glusterd2/commands/volumes/volume-option.go

```diff
@@ -206,6 +206,11 @@ func volumeOptionsHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if containsReservedGroupProfile(req.Options) {
```

Comment:
do we need this check during volume create also?

Reply:
makes sense. I will add it during create as well

Reply:
done